scitail/README.md

```diff
@@ -0,0 +1,27 @@
+# SciTail
+
+* [evaluator](evaluator/) is the program used by the AI2 Leaderboard to evaluate submitted predictions.
+
+## Example usage
+
+To evaluate dummy predictions (every pair of sentences is predicted to entail) against the SciTail dataset, run this:
+
+```
+% python3 evaluator/evaluator.py -a data/test/answers.jsonl -p data/test/predictions.csv -o metrics.json 
+
+% cat metrics.json
+{"accuracy": 0.39604891815616183}
+```
+
+Replace `data/test/predictions.csv` with your predictions to compute your test score. 
```

Comment:
Same here.

Reply:
Right. Thanks for catching these.

Both are fixed now.